config/typo3-95.yaml

```diff
@@ -5,10 +5,19 @@ services:
     Ssch\TYPO3Rector\Rector\Fluid\ViewHelpers\MoveRenderArgumentsToInitializeArgumentsMethodRector: ~
     Rector\Php55\Rector\String_\StringClassNameToClassConstantRector: ~
     Ssch\TYPO3Rector\Rector\Annotation\InjectAnnotationRector: ~
-    Ssch\TYPO3Rector\Rector\Annotation\CascadeAnnotationRector: ~
     Ssch\TYPO3Rector\Rector\Annotation\IgnoreValidationAnnotationRector: ~
-    Ssch\TYPO3Rector\Rector\Annotation\LazyAnnotationRector: ~
-    Ssch\TYPO3Rector\Rector\Annotation\TransientAnnotationRector: ~
+
+    Ssch\TYPO3Rector\Rector\Annotation\ReplaceAnnotationRector:
+        $oldToNewAnnotations:
+            # @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Feature-83078-ReplaceLazyWithTYPO3CMSExtbaseAnnotationORMLazy.html
+            lazy: 'TYPO3\CMS\Extbase\Annotation\ORM\Lazy'
+
+            # @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Deprecation-83093-ReplaceCascadeWithTYPO3CMSExtbaseAnnotationORMCascade.html
+            cascade: 'TYPO3\CMS\Extbase\Annotation\ORM\Cascade("remove")'
+
+            # @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.0/Feature-83092-ReplaceTransientWithTYPO3CMSExtbaseAnnotationORMTransient.html
+            transient: 'TYPO3\CMS\Extbase\Annotation\ORM\Transient'
+
```

Comment:
If there a pattern, like replace annotation with another one, a configured Retor can be used.

In this case, I've decopuled `ReplaceAnnotationRector` to be used in all other cases repeatedly